src/screens/Requests/Requests.spec.tsx

```
@@ -19,61 +19,67 @@ import {
-// Ensure window.localStorage reflects the stub in JSDOM
-Object.defineProperty(window, 'localStorage', {
-  value: localStorageMock,
-  configurable: true,
-  writable: true,
 });
 
+// We don't need to mock global localStorage anymore for the hook
+// But we might need it if the component uses localStorage directly
+// The component uses useLocalStorage hook.
+
+const { setItem, removeItem } = useLocalStorage();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)